iib/web/static/api_v1.yaml

```diff
@@ -752,6 +752,11 @@ components:
             index images that IIB does not have access to. The format of the token is in the format
             "user:password".
           example: token
+        registry_auths:
```

Comment:
@lcarva which iib operations should have registry_auths added?  Add operator bundle, remove operator, anything else?

Reply:
That's a good question. It's needed in at least `regenerate-bundle`. Let's follow up with the CVP on this.

Reply:
Confirmed with them. Let's just add it to `regenerate-bundle`.